codeGenerator/src/FRCrobotCodeGen302/MainForm.cs

```diff
@@ -39,11 +39,26 @@ public MainForm()
             valueTextBox.Location = valueNumericUpDown.Location;
 
             this.Text += " Version " + ProductVersion;
+
+            //try to load cached configuration.xml
+            addProgress("Trying to load cached configuration.xml");
+            if (File.Exists(Path.GetTempPath() + "DragonsCodeGeneratorCache.txt"))
+            {
+                string cachedPath = File.ReadAllText(Path.GetTempPath() + "DragonsCodeGeneratorCache.txt");
```

Comment:
lines 47 to 50 need to be in a 
try
{
 ... 
}
catch(Exception ex)
            {
                addProgress("Issue encountered while loading the generator configuration file\r\n" + ex.ToString());   
            }